service.partymode.autostart/addon.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.partymode.autostart"
+       name="Partymode autostart"
+       version="0.9.2"
+       provider-name="muttley">
```

Comment:
Are you the original author? If not add yourself as an author by keeping the credits

Reply:
Yes, I'm. I use two nickname randomly.
You can check my github profile...anyway I have update this field